contracts/erc721/OpenSeaGasFreeListing.sol

```diff
@@ -19,9 +31,27 @@ library OpenSeaGasFreeListing {
         view
         returns (bool)
     {
-        ProxyRegistry registry;
+        uint256 chainId = getChainId();
+        if (chainId == 1 || chainId == 4) {
+            return isApprovedForAllWyvern(owner, operator, chainId);
+        } else {
```

Comment:
Please add the explicit chain IDs here to avoid running the ZeroEx branch on other chains.